test/test.go

```diff
@@ -17,6 +19,12 @@ type Parameter struct {
 	Length   int
 }
 
+type Node struct {
```

Comment:
Obrigado por isso @davidsmedeiros 💯  . Sugestão pra ficar mais claro o objetivo 😀 
```suggestion
type XmlNode struct {
```